charts/all/config-demo/templates/config-demo-secret.yaml

```diff
@@ -0,0 +1,67 @@
+{{ if .Values.clusterGroup.isHubCluster }}
```

Comment:
This is more than a little counter-intuitive, I fear.  The code is absolutely correct (since the policy can only be instantiated on the hub cluster), and I was about to ask you to check it, until I thought it through myself.

Reply:
Agreed, let me add a comment so our future selves will be less confused